### PR TITLE
Updated Braintree connector docs

### PR DESCRIPTION
**Pull request by sandip-baidya**
Hi,
Followed Amruta's youTube video's reference to edit Braintree's connector docs. This is my first open-source contribution!
1. Added prerequisites section
2. consistent header sizes
3. Added set up guide (similar to Freshdesk and Greenhouse)

Following are some of the areas I did not touch because there was ambiguity there and I did not want to change anything around because of my limited knowledge of both Airbyte and Braintree:-
1. The sentence "creating read-only key specifically for Airbyte access" could do well with specifying what key are we talking about exactly because there are two keys; public and private and I couldn't really find that out on my own either from google. (So, perhaps someone can edit that in the future)
2. Sync Overview (I was not sure about whether all 4 kinds of syncs are being supported for Braintree from what was originally written in the docs, so I let the original sentence be as is; did not modify it one bit)
3. Data type mapping (Again, I was not sure what data mapping truly meant here, so I let it be, as is)

Thanks.

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@Amruta-Ranade can you check this contribution?

**Review by Amruta-Ranade — CHANGES_REQUESTED** [review]
Requires a tiny change, but overall, looks good! Thank you for your contribution!

**Comment by sandip-baidya** [comment]
> Requires a tiny change, but overall, looks good! Thank you for your contribution!

Hi,
I'm new to this. Just wanted to know if I have to do anything to accommodate this change you've requested, or whether that is something that you can do from your end because I couldn't exactly find the change you requested. (I know you're busy, so take your time and please let me know what the next steps are, once you're available ::)

Also, I can follow the guidelines from your video and edit all the connector docs one by one (especially the ones that have not gone through any edits at all)

File: docs/integrations/sources/braintree.md [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental Sync | Yes |  |
+| Namespaces | No |  |
+
+## Supported Streams
 
-This Source is capable of syncing the following core Streams:
+Several output streams are available from this source:
```
Comment (Amruta-Ranade) [inline docs/integrations/sources/braintree.md:43]: Rephrase as "The following streams are supported:"

**Comment by Amruta-Ranade** [comment]
> > Requires a tiny change, but overall, looks good! Thank you for your contribution!
> 
> Hi, I'm new to this. Just wanted to know if I have to do anything to accommodate this change you've requested, or whether that is something that you can do from your end because I couldn't exactly find the change you requested. (I know you're busy, so take your time and please let me know what the next steps are, once you're available ::)
> 
> Also, I can follow the guidelines from your video and edit all the connector docs one by one (especially the ones that have not gone through any edits at all)

Ah, sorry, the comment didn't save yesterday. Let me know if you see it now!
We'd love your help with other connectors. Let's merge this PR and discuss what you can pick up next :)

**Comment by sandip-baidya** [comment]
Hello,
Addressed your changes. Please review once from your side. And if that gets successfully done, then let's talk about other connector docs ::)

**Comment by sandip-baidya** [comment]
@Amruta-Ranade hello. Just wanted to check if my pull request went through successfully and if not, am I required to do something else for it to go through properly?

PS I could not figure out how to incorporate the changes you requested in the original way I edited this document (i.e. straight into GitHub) so I ended up using git to push changes from my local system, onto GitHub and I'm worried that it may not have sent properly)

**Review by Amruta-Ranade — APPROVED** [review]

**Comment by Amruta-Ranade** [comment]
> @Amruta-Ranade hello. Just wanted to check if my pull request went through successfully and if not, am I required to do something else for it to go through properly?
> 
> PS I could not figure out how to incorporate the changes you requested in the original way I edited this document (i.e. straight into GitHub) so I ended up using git to push changes from my local system, onto GitHub and I'm worried that it may not have sent properly)

Thanks for the ping! The PR looks good. Can you sign the CLA with the new user id you used to push the latest change?

**Comment by sandip-baidya** [comment]
Hello!
I am not able to sign the CLA with the new user. Turns out, when I pushed changes from my local machine, I had not explicitly linked my GitHub email with that user. Can I just open a new PR with all of the changes done till now, and delete this particular PR?

(Or I could make tiny changes, for example, removal of space or something, and push again with a user that's linked to my actual GitHub account?) Apologies for the so-many back and forth.

**Comment by Amruta-Ranade** [comment]
> Hello! I am not able to sign the CLA with the new user. Turns out, when I pushed changes from my local machine, I had not explicitly linked my GitHub email with that user. Can I just open a new PR with all of the changes done till now, and delete this particular PR?
> 
> (Or I could make tiny changes, for example, removal of space or something, and push again with a user that's linked to my actual GitHub account?) Apologies for the so-many back and forth.

I asked the team what's the best course of action here. Will let you know when I hear back :)

**Comment by Amruta-Ranade** [comment]
The team recommended the second option: Make a small change and push with the correct username

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**